src/js/jsx/DocumentHeader.jsx

```diff
@@ -31,14 +31,19 @@ define(function (require, exports, module) {
         Immutable = require("immutable");
 
     var os = require("adapter/os");
-    
-    var DocumentHeaderTab = require("jsx!js/jsx/DocumentHeaderTab");
+
+    var DocumentHeaderTab = require("jsx!js/jsx/DocumentHeaderTab"),
+        Button = require("jsx!js/jsx/shared/Button"),
+        SVGIcon = require("jsx!js/jsx/shared/SVGIcon"),
+        strings = require("i18n!nls/strings");
 
     var DocumentHeader = React.createClass({
         mixins: [FluxMixin, StoreWatchMixin("application", "document")],
 
         getInitialState: function () {
-            return {};
+            return {
+                exportDisabled: true
```

Comment:
I think you can actually just remove exportDisabled from the react state (here, and `componentWillReceiveProps` in its entirety).  It isn't actually being used (you are deriving it within `render()` separately,  and that's totally sufficient)